docs/reference/rfcs/FEEDBACK-0001-release-automation.md

```
@@ -0,0 +1,10 @@
+Love the direction. A couple of places to sharpen before we merge:
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)